lib/lighthouse/benefits_documents/configuration.rb

```diff
@@ -93,7 +93,8 @@ def generate_upload_body(document_data, file_body)
       file = Tempfile.new(document_data[:file_name])
       File.write(file, file_body)
 
-      payload[:file] = Faraday::UploadIO.new(file, file_type)
+      file_mime_type = MimeMagic.by_path(document_data[:file_name]).type
+      payload[:file] = Faraday::UploadIO.new(file, file_mime_type)
```

Comment:
mixed up the `subtype` and `type`. subtype is the shorthand (jpg) while type is the actual mime type that the file needs (image/jpg).